### PR TITLE
Update literals.md

### PR DESCRIPTION
**Pull request by cartermp**
Reshuffle some stuff and clarify examples

### CODE REVIEW

File: docs/fsharp/language-reference/literals.md [hunk 1]
```diff
@@ -38,13 +38,16 @@ The following table shows the literal types in F#. Characters that represent dig
 |byte[]|ASCII string|B|`"text"B`|
 |String or byte[]|verbatim string|@ prefix|`@"\\server\share"` (Unicode)<br /><br />`@"\\server\share"B` (ASCII)|
 
-## Remarks
+## Named Literals
```
Comment (mairaw) [inline docs/fsharp/language-reference/literals.md]:
```suggestion
## Named literals
```

File: docs/fsharp/language-reference/literals.md [hunk 2]
```diff
@@ -58,13 +61,11 @@ let Literal2 = 1 ||| 64
...
-## Named Literals
+## Remarks
 
-Values that are intended to be constants can be marked with the [Literal](https://msdn.microsoft.com/library/465f36ce-d146-41c0-b425-679c509cd285) attribute. This attribute has the effect of causing a value to be compiled as a constant.
+Unicode strings can contain explicit encodings that you can specify by using `\u` followed by a 16-bit hexadecimal code or UTF-32 encodings that you can specify by using `\U` followed by a 32-bit hexadecimal code that represents a Unicode surrogate pair.
 
-In pattern matching expressions, identifiers that begin with lowercase characters are always treated as variables to be bound, rather than as literals, so you should generally use initial capitals when you define literals.
+The use of other bitwise operators other than `|||` isn't allowed.
 
 ## Integers In Other Bases
```
Comment (mairaw) [inline docs/fsharp/language-reference/literals.md]:
```suggestion
## Integers in other bases
```

File: docs/fsharp/language-reference/literals.md [hunk 3]
```diff
@@ -77,7 +78,7 @@ let numbers = (0x9F, 0o77, 0b1010)
 
 ## Underscores in Numeric Literals
```
Comment (mairaw) [inline docs/fsharp/language-reference/literals.md]:
```suggestion
## Underscores in numeric literals
```

**Review by mairaw — APPROVED** [review]
Changes look good. Left some suggestions for you to consider. Plus, should you update ms.date?

**Review by rpetrusha — APPROVED** [review]
LGTM, @cartermp. You can merge when you're ready.

**Pull request merged**